openvpn/start.sh

```diff
@@ -245,7 +245,13 @@ function ufwAllowPort {
   portNum=${1}
   if [[ "${ENABLE_UFW,,}" == "true" ]] && [[ -n "${portNum-}" ]]; then
     echo "allowing ${portNum} through the firewall"
-    ufw allow ${portNum}
+    if [[ $portNum == *":"* ]];
+    then
+      ufw allow ${portNum}/tcp
```

Comment:
So this is required to allow a range of ports eh?
Thanks for the fix

Reply:
Yes, or if you don't like IF statement we can just use:
```
ufw allow ${portNum}/tcp
ufw allow ${portNum}/udp
```
instead of:
```
ufw allow ${portNum}
```

Correct command for range of ports is:
```
ufw allow ###:####/tcp
ufw allow ###:####/udp
```
where one port can be added using `ufw allow ###` but for one port command `ufw allow ###/tcp` is also working.